pom.xml

```diff
@@ -1037,7 +1037,7 @@
 		<aspectj.version>1.8.1</aspectj.version>
 		<hibernate.validator.version>5.1.1.Final</hibernate.validator.version>
 		<el-ri.version>1.0</el-ri.version>
-		<com.github.jmchilton.blend4j.version>0.2.0-SNAPSHOT-29f28db211c300fcaf21a7a5992478fc5a9893cd</com.github.jmchilton.blend4j.version>
+		<com.github.jmchilton.blend4j.version>0.2.0-SNAPSHOT-044e471c96ebe1d97ccff3abceab047ffd36a660</com.github.jmchilton.blend4j.version>
```

Comment:
So I'm adding the new blend4j library directly into IRIDA at http://gitlab-irida.corefacility.ca/irida/irida/merge_requests/1202.

Once that's merged into the branch `development` could you merge `development` into this branch and update the blend4j version to `0.2.0-SNAPSHOT-fac9057d047377e679b6c14672b364a20b9a462b`?

Reply:
Ok, will do.

Reply:
Done.

Reply:
Thanks.